```
/home/runner/work/mod-mall-teleport/mod-mall-teleport/modules/mod-mall-teleport/src/mod-mall-teleport.cpp:94:35: fatal error: no member named 'SendNotification' in 'WorldSession'
   94 |             player->GetSession()->SendNotification("You do not have access to this command");
      |             ~~~~~~~~~~~~~~~~~~~~  ^
1 error generated.
gmake[2]: *** [modules/CMakeFiles/modules.dir/build.make:107: modules/CMakeFiles/modules.dir/mod-mall-teleport/src/mod-mall-teleport.cpp.o] Error 1
```

--- a/src/mod-mall-teleport.cpp
+++ b/src/mod-mall-teleport.cpp
@@ -91,7 +91,7 @@ public:
 
         if (!enabled)
         {
-            player->GetSession()->SendNotification("You do not have access to this command");
+            ChatHandler(player->GetSession()).SendNotification("You do not have access to this command");
             return false;
         }
 

